pkg/builddef/build_opts.go

```diff
@@ -57,3 +57,20 @@ func LockFilepath(ymlFile string) string {
 	ext := filepath.Ext(ymlFile)
 	return strings.TrimSuffix(ymlFile, ext) + ".lock"
 }
+
+// UpdateLocksOpts represents the parameters passed to specialized builders to
+// update a lockfile.
+// @TODO: move to builder pkg.
```

Comment:
should this be done now ?

Reply:
Unfortunately it's not possible to move this struct to builder package since the defkind `registry` package needs this struct to define the defkind handler interface and the `builder` package needs the `registry` package :slightly_frowning_face: